Quantum-Control-Applications/Superconducting/Multiple Flux Tunable Transmons/Use Case 2 - Two Qubit Randomized Benchmarking/README.md

```diff
@@ -0,0 +1,238 @@
+
+### The TwoQubitRB Class
+```python
+rb = TwoQubitRb(config, single_qubit_gate_generator, two_qubit_gate_generators, prep_func, measure_func, verify_generation=True)
+```
+
+- **TwoQubitRb**: The class for generating the configuration and running two-qubit randomized benchmarking experiments with the OPX
+- **config**: dict – Standard configuration “config” containing the relevant experimental details (e.g. what analog outputs are connected to the xy drive, z flux line, etc.).
+- **single_qubit_gate_generator**: A callable used to generate a generic (baked) single qubit gate using a signature similar to phasedXZ
+- **two_qubit_gate_generators**: Mapping two qubit gate names to callables used to generate the (baked) gates (needs at least one two-qubit gate). Can contain all two-qubit gates implemented by the user.
+- **prep_func**: Callable used to reset the qubits to the |00> state. This function does not use the baking object, and is a proper QUA code macro (e.g. wait() statement or active reset protocol).
+- **measure_func**: A callable used to measure the qubits. This function does not use the baking object, and is a proper QUA code macro. Returns a tuple containing the measured values of the two qubits as QUA expressions. The expression must evaluate to a Boolean value. False means |0>, True means |1>. The most significant bit (MSB) is the first qubit. 
+verify_generation: bool = False
+
+### Run the 2 QB RB 
+The experiment is run by calling the run method of the previously generated program rb.
+
+```python
+qmm = QuantumMachinesManager('127.0.0.1',8080)
+res = rb.run(qmm, circuit_depths=[1, 2, 3, 4, 5], num_circuits_per_depth=50, num_shots_per_circuit=1000)
+```
+
+For running the experiment the user has to specify the following arguments:
+- **qmm**: The quantum machine manager instance, on which the 2-Qubit-RB will be executed on.
+- **circuit_depths**: Number of consecutive clifford gates (layers) per sequence (not including the inverse.
+- **num_circuits_per_depth**: The amount of different circuit randomizations (combination of Cliffords) in each sequence. 
+- **num_shots_per_circuit**: The number of repetitions of the same circuit of a depth, e.g. used for averaging.
+
+### Gate Definition
+Gate generation is performed using the *baking* class. This class adds to QUA the ability to generate arbitrary waveforms ("baked waveforms") using syntax similar to QUA. 
+#### single_qubit_gate_generator
+```python
+def bake_phased_xz(baker: Baking, q, x, z, a):
+    element = f"qubit{q}_xy"
+    baker.frame_rotation_2pi(-a, element)
+    baker.play("x", element, amp=x)
+    baker.frame_rotation_2pi(a + z, element)
+```
+single_qubit_gate_generator: A callable used to generate a single qubit gate using a signature similar to `phasedXZ`.
+Callable arguments:  
+- **baking**: The baking object. 
+- **qubit**: The qubit number. 
+- **x**: The x rotation exponent. 
+- **z**: The z rotation exponent. 
+- **a**: the axis phase exponent. 
```

Comment:
Maybe add the reference to: https://quantumai.google/reference/python/cirq/PhasedXZGate